.github/workflows/multi-arch-gating.yaml

```diff
@@ -0,0 +1,66 @@
+name: Mult-Arch-Gating
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build-image:
+    name: Build Cachi2 image and run smoketest on it
+    runs-on: ubuntu-latest
+    steps:
+      - name: add checkout action...
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          
+      - name: Install required packages
+        run: |
+          python -m pip install --upgrade pip
+          pip install tox tox-gh-actions
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+
+      - name: Build Cachi2 image
+        uses: redhat-actions/buildah-build@v2
+        with:
+          image: cachi2
+          tags: ${{ github.sha }}
+          platforms: linux/amd64,linux/ppc64le
+          containerfiles: |
+            ./Containerfile
+      - name: Check image created and Cachi2 version
+        run: |
+          buildah images | grep 'cachi2'
+          podman run -t cachi2:${{ github.sha }} --version
+      - name: Run smoketest on built images
+        run: |
+          git clone -b v2.1.1 https://github.com/release-engineering/retrodep.git /tmp/source
+          podman run -t -v /tmp/source:/tmp/source \
+            cachi2:${{ github.sha }} \
+            fetch-deps \
+            --source /tmp/source \
+            --output ./cachi2-output \
+            --package '{"path": ".", "type": "gomod"}'
+      - name: Run integration tests on built image
+        env:
+          CACHI2_IMAGE: localhost/cachi2:${{ github.sha }}
+        run: |
+           tox -e integration
```

Comment:
I think the idea was for this workflow to be _just_ an image build, no tests necessary. I would drop integration tests from here for sure, even the smoketest is probably not necessary

Reply:
I have removed the tests. @chmeliik can u check it?